README-sandbox.md

```diff
@@ -0,0 +1,190 @@
+operator, another for the block producer and yet another for the SNARK
+worker. Let's put them in `keys` directory:
+
+```shell
+$ mkdir -p keys
+```
+
+Before creating the keys it makes sense to set up some environment
+variables:
+
+```shell
+$ export MINA_LIBP2P_PASS=
+$ export MINA_PRIVKEY_PASS=
+```
+
+These variables hold passwords protecting private keys. If they are
+not set, the node will ask for those passwords before creating the
+keys. It won't be able to access these keys without these variables
+set. User is free to put any passwords there, however, in development
+or testing setups it not really necessary, as those networks don't
+hold any real assets anyway. These settings are essential for the
+security of mainnet nodes, though, and that's why they're mandatory.
+
+```shell
+$ mina libp2p generate-keypair --privkey-path keys/node.key
+```
+
+This command creates a key for the node operator. This is a key to use
+to identify the node in the p2p network. In a single-network setup it
+might not be the most relevant, but it's required nonetheless.
+
+Then we need key pairs for the block producer(s) and SNARK worker(s):
+
+```shell
+$ mina advanced generate-keypair --privkey-path keys/block-producer.key
+$ mina advanced generate-keypair --privkey-path keys/snark-producer.key
+$ chmod -R 0700 keys
+```
+
+Don't forget to set key files' permissions to `0700` or else the
+client will refuse to import them.
+
+Additionally, the block producer's key should be copied to the `wallet`
+directory in the node's config dir. This directory doesn't exist yet
+probably, so one can create it by hand or try launching the node to
+set up the config directory. The filename should be identical to the
+block producer's public key.
+
+```shell
+$ mkdir -p .mina-config/wallet/store
+$ cp keys/block-producer.key .mina-config/wallet/store/<private-key>
+```
+
+Of course we are free to produce more keys for regular users of the
+network or if we wish to have more nodes/block producers/snarkers
+etc. Note that we need to generate those keys before the next step if
+we wish to give those accounts some initial balance. Otherwise we
+might as well create them later.
+
+The Genesis Ledger
+--------------
+
+A skeleton genesis ledger looks like this:
+
+```genesis-ledger.json
+{
+  "genesis": {
+    "genesis_state_timestamp": "2022-10-20T12:00:00Z"
+  },
+  "ledger": {
+    "name": "sandbox",
+    "accounts": [ ]
+  }
+}
+```
+
+Now we need to populate the `.ledger.accounts` section with users of
+the network that we want to hold some initial balance. At the very
+least we need to put block producer's account here so that he has a
+stake necessary to produce blocks. Each account in the genesis ledger
+is represented by an object with the following data:
+
+```genesis-ledger.json
+      {
+        "pk": "<the public key>",
+        "balance": "<the amount of mina as string with up to 9 decimal places>",
+        "delegate": "<the pub key of the delegate of null",
```

Comment:
```suggestion
        "delegate": "<the pub key of the delegate or null>",
```
Maybe it can be added that the delegate is usually the block producer